apps/web-twig-demo/templates/layout/base.html.twig

```diff
@@ -3,7 +3,8 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
-    <title>Spirit Web Twig — Development Preview</title>
+
+    <title>{% block title %}{% endblock%} — Web Twig — Spirit Design System</title>
```

Comment:
are we sure the title is everywhere? Maybe the — can be rendered using some condition?